test/integration/agent_policy_simple_example/controls/gcloud.rb

```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 Google LLC
+# Copyright 2020 Google LLC
```

Comment:
Can we verify the policy was actually created?

Reply:
Yep, you're totally right!